.github/workflows/main.yml

```diff
@@ -298,6 +298,10 @@ jobs:
 
       - name: run itest
         run: make itest-parallel windows=1
+        
+      - name: kill any remaining lnd processes
+        if: ${{ failure() }}
+        run: killall --wait lnd-itest.exe
```

Comment:
there is a `make itest-clean` command you could use tho, if it helps.

Reply:
Ah, yes, I forgot about that, thanks. But it looks like the `killall` command isn't available on Windows. I've been trying a whole bunch of things... Windows is so weird...